sqlglot/tokens.py

```diff
@@ -123,6 +130,7 @@ class TokenType(AutoName):
     CURRENT_TIME = auto()
     CURRENT_TIMESTAMP = auto()
     DIV = auto()
+    DECLARE = auto()
```

Comment:
why did you add this it's not used?

Reply:
Yeah I'll remove it, I was trying out the declare syntax and forgot it. We can add it again later if needed.